app_feup/lib/view/Widgets/info_desk_card.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/material.dart';
+import 'generic_card.dart';
+
+/// Manages the 'Current account' section inside the user's page (accessible
+/// through the top-right widget with the user picture)
+class InfoDeskCard extends GenericCard {
+  InfoDeskCard({Key key}) : super(key: key);
+
+  InfoDeskCard.fromEditingInformation(
+      Key key, bool editingMode, Function onDelete)
+      : super.fromEditingInformation(key, editingMode, onDelete);
+
+  @override
+  Widget buildCardContent(BuildContext context) {
+    return Column(children: [
+      Table(
+          defaultVerticalAlignment: TableCellVerticalAlignment.middle,
+          children: [
+            TableRow(children: [
+              Container(
+                margin: const EdgeInsets.only(
+                    top: 20.0, bottom: 0.0, left: 20.0), // add margin left: 20
+                child: Text('Horários',
+                    //textAlign: TextAlign.center,
+                    style: Theme.of(context)
+                        .textTheme
+                        .headline4
+                        .apply(fontSizeDelta: 0)),
+              )
+            ]),
+            TableRow(children: [
+              Container(
+                margin: const EdgeInsets.only(
+                    top: 13.0, bottom: 0.0, left: 20.0), // add margin left: 20
+                child: Text('Atendimento presencial e telefónico',
+                    //textAlign: TextAlign.center,
+                    style: Theme.of(context)
+                        .textTheme
+                        .headline4
+                        .apply(fontSizeDelta: -4)),
+              )
+            ]),
+            TableRow(children: [
+              Container(
+                  margin:
+                      const EdgeInsets.only(top: 8, bottom: 12.0, left: 20.0),
+                  child: Text('9:30h - 13:00h | 14:00h - 17:30h',
+                      //textAlign: TextAlign.center,
+                      style: Theme.of(context).textTheme.headline3))
+            ]),
```

Comment:
Personally, I think the widgets organization is a bit confusing. Instead of using a Table inside a Column and a set of TableRow's for the lines containing the necessary info, I think it would be more simple and readable to just use a Column of Containers. For exemple, something like this (which can also be done in the academic_services_card). What do you think?

```suggestion
  Widget buildCardContent(BuildContext context) {
    return Column(
      children: <Container> [
        Container(
          margin: const EdgeInsets.only(top: 20.0, bottom: 0.0, left: 20.0),
          child: Align(
            alignment: Alignment.centerLeft,
            child: Text('Horários',
                style: Theme.of(context)
                    .textTheme.headline4
                    .apply(fontSizeDelta: 0)),
          ),
        ),
        Container(
          margin: const EdgeInsets.only(top: 13.0, bottom: 0.0, left: 20.0),
          child: Align(
            alignment: Alignment.centerLeft,
            child: Text('Atendimento presencial e telefónico',
                style: Theme.of(context)
                    .textTheme.headline4
                    .apply(fontSizeDelta: -4)),
          ),
        ),
        Container(
          margin: const EdgeInsets.only(top: 8, bottom: 12.0, left: 20.0),
          child: Align(
            alignment: Alignment.centerLeft,
            child: Text('9:30h - 13:00h | 14:00h - 17:30h',
                style: Theme.of(context)
                    .textTheme.headline3),
          ),
        ),
      ],
```